modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -179,6 +179,14 @@ enum class Feature(
         hasFirebaseRemoteFlag = true,
         hasDevToggle = false,
     ),
+    BASIC_AUTHENTICATION(
+        key = "basic_authentication",
+        title = "Support basic authentication when downloading or streaming episodes",
+        defaultValue = BuildConfig.DEBUG,
+        tier = FeatureTier.Free,
+        hasFirebaseRemoteFlag = true,
```

Comment:
I have linked this to a Firebase Config parameter in case we need to turn it off if we have an issue in production.